public/angular/translations/locale-en.json

```diff
@@ -392,11 +392,22 @@
   "Whatsapp": "WhatsApp",
   "whatsapp": "WhatsApp",
   "gplaces": "Google Places",
+  "CrowdTangle": "crowdtangle",
+  "crowdtangle" : "CrowdTangle",
+  "CrowdTangle": "CrowdTangle",
+  "Crowdtangle" : "CrowdTangle",
```

Comment:
Why are there four different capitalization's of "crowdtangle" here?

Reply:
Would clean that up. I was testing stuff a while ago.